lib/Twig/Extensions/Extension/Intl.php

```diff
@@ -131,5 +132,15 @@ function twig_get_number_formatter($locale, $style)
 
     $formatter = NumberFormatter::create($locale, $styleValues[$style]);
 
+    foreach ($attributes as $name => $value) {
+        $constantName = strtoupper($name);
+
+        if (!defined('NumberFormatter::'.$constantName)) {
+            throw new Twig_Error_Syntax(sprintf('NumberFormatter has no attribute "%s"', $name));
```

Comment:
The other exceptions in the file don't have the dot at the end... So I kept this (probably unwanted) convention too.